plugins/wasi_ocr/Makefile

```diff
@@ -0,0 +1,307 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
I got so confused about this file. It would be best if you never committed Makefile to this project since we already use cmake to manage the building process. This file also contains many of your local paths, which are also unavailable to others.

Reply:
I apologize for not checking correctly, both this file and the `cmake_install.cmake` are automatically generated files by CMake and should not have been committed, and somehow skipped the gitignore. I'll remove them in the next commit.